### DISCUSSION TITLE
Missing Product List Endpoints

### DISCUSSION THREAD

**Discussion by joshistoast** [Shopify/storefront-api-feedback#85] [Ideas & Requests]
Would love to see more and more data available from liquid be available to the storefront api, a few pressing ones for me and my company's store would be:
- [allProductsCount](https://shopify.dev/api/liquid/objects/collection#collection-all_products_count)
- [productsCount](https://shopify.dev/api/liquid/objects/collection#collection-products_count)
- [allTags](https://shopify.dev/api/liquid/objects/collection#collection-all_tags)
- [allTypes](https://shopify.dev/api/liquid/objects/collection#collection-all_types)

These would help greatly for creating better sorting and filtering of products in collections. These would also come in handy not just for collections, retrieving products outside of collections with these endpoints would be great for more dynamic site-wide search functionality.

**Comment by patjakubik** [comment]
Storefront API generally lacks "counts". Similar issue here, but regarding orders count for customer accounts: https://github.com/Shopify/storefront-api-feedback/discussions/83

**Comment by designifyer** [comment]
+1
